src/app/server.js

```diff
@@ -206,20 +208,22 @@ function runApp () {
 
   // redirect to home if nothing found
   app.all('*', (req, res) => res.redirect('/'))
-
-  portal.server = app.listen(envHelper.PORTAL_PORT, () => {
-    if (envHelper.PORTAL_CDN_URL) {
-      const req = request
-        .get(envHelper.PORTAL_CDN_URL + 'index.' + packageObj.version + '.' + packageObj.buildHash + '.ejs')
-        .on('response', function (res) {
-          if (res.statusCode === 200) {
-            req.pipe(fs.createWriteStream(path.join(__dirname, 'dist', 'index.ejs')))
-          } else {
-            console.log('Error while fetching '+envHelper.PORTAL_CDN_URL + 'index.' + packageObj.version + '.' + packageObj.buildHash + '.ejs file when CDN enabled');
-          }
-        })
-    }
-    console.log('app running on port ' + envHelper.PORTAL_PORT)
+  // start server after fetching the configuration data
```

Comment:
what happens if a error is there in the call will the server be started?

Reply:
If there is an error it will fallback to environment variables configuration and this is handled in configHelper.